tests/ci/cdk/cdk/codebuild/github_ci_linux_arm_omnibus.yaml

```diff
@@ -262,6 +262,29 @@ batch:
         variables:
           AWS_LC_CI_TARGET: "tests/ci/run_fips_tests.sh"
 
+    - identifier: amazonlinux2_clang7x_aarch_fips_callback
```

Comment:
Why test clang7 for AL2 in ARM, but not for x86?

Reply:
Good catch this should be gcc 7 to match what's used by default.